software/coreip-welcome/coreip-welcome.c

```diff
@@ -0,0 +1,156 @@
+
+#include <stdio.h>
+#include <metal/cpu.h>
+#include <metal/led.h>
+#include <metal/button.h>
+#include <metal/switch.h>
+
+#define RTC_FREQ    32768
+
+struct metal_cpu *cpu0;
+struct metal_interrupt *cpu_intr, *tmr_intr;
+int tmr_id;
+volatile uint32_t timer_isr_flag;
+
+void display_banner (void) {
+
+    printf("\n");
+    printf("\n");
+    printf("                  SIFIVE, INC.\n");
+    printf("\n");
+    printf("           5555555555555555555555555\n");
+    printf("          5555                   5555\n");
+    printf("         5555                     5555\n");
+    printf("        5555                       5555\n");
+    printf("       5555       5555555555555555555555\n");
+    printf("      5555       555555555555555555555555\n");
+    printf("     5555                             5555\n");
+    printf("    5555                               5555\n");
+    printf("   5555                                 5555\n");
+    printf("  5555555555555555555555555555          55555\n");
+    printf("   55555           555555555           55555\n");
+    printf("     55555           55555           55555\n");
+    printf("       55555           5           55555\n");
+    printf("         55555                   55555\n");
+    printf("           55555               55555\n");
+    printf("             55555           55555\n");
+    printf("               55555       55555\n");
+    printf("                 55555   55555\n");
+    printf("                   555555555\n");
+    printf("                     55555\n");
+    printf("                       5\n");
+    printf("\n");
+
+    printf("\n");
+    printf("Welcome to the E31/E51 Core IP FPGA Evaluation Kit!\n");
```

Comment:
Best to start avoid specific coreIP type, since the number of variant now increases.
May be something like "printf("Welcome to the Standard Core IP FPGA Evaluation Kit!\n");"
Maybe freedom-metal should have an API to return this string

Reply:
Ok I fixed the welcome banner message and updated the submodule - thanks